sql/5_0_2-to-6_0_0_upgrade.sql

```diff
@@ -761,3 +761,79 @@ ALTER TABLE `facility` ADD `uuid` binary(16) DEFAULT NULL;
 #IfNotIndex facility uuid
 CREATE UNIQUE INDEX `uuid` ON `facility` (`uuid`);
 #EndIf
+
+#IfNotRow codes code 145 code_type 100
```

Comment:
note we can't assume the code_type (this is flexible and will be different for upgraders where it was initiallt added in the sql/4_0_0-to-4_1_0_upgrade.sql script)

Reply:
ok, do we also need to have an `IfNotTable` and then import them all?